[2.0] Describe AssetTargetFallback

Similar to the request to surface `AssetTargetFallback` in https://github.com/NuGet/docs.microsoft.com-nuget/issues/410, which was prompted by a discussion regarding `PackageTargetFallback` [here](https://github.com/dotnet/cli/issues/3439#issuecomment-319702662), there are a few topics over here that might (probably) need a smidgen of work ... a note? a section? a replacement? versioned content? ... something to surface it:

https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagetargetfallback
https://docs.microsoft.com/en-us/dotnet/core/tools/project-json-to-csproj#imports

Cross ref: [Documenting .NET Core MSBuild targets and properties/CSPROJ (2642)](https://github.com/dotnet/docs/issues/2642)

Note that especially when upgrading to 2.0, users might see this error if they have been using `PackageTargetFallback `:

```
error NU1003: PackageTargetFallback and AssetTargetFallback cannot be used together. Remove PackageTargetFallback(deprecated) references from the project environment.
```
see https://github.com/dotnet/cli/issues/6952.
TL;DR unless `DisableImplicitAssetTargetFallback` is set to `false`, .NET Core and .NET Standard >= 2.0 projects will implicitly have their `AssetTargetFallback` set to include `net461` (see [source](https://github.com/dotnet/sdk/blob/cd25d6bbfc2aac61cc790c511900427ac64fdaa8/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets#L139-L141)). 

The only docs I found are in a table at the bottom of https://github.com/NuGet/Home/wiki/Enable-.NET-Core-2.0-projects-to-work-with-.NET-Framework-4.6.1-compatible-packages

--- a/docs/core/project-sdk/msbuild-props.md
+++ b/docs/core/project-sdk/msbuild-props.md
@@ -148,6 +148,21 @@ The `PackageReference` item lets you specify a NuGet dependency. For example, yo
 
 For more information, see [Package references in project files](/nuget/consume-packages/package-references-in-project-files).
 
+### AssetTargetFallback
+
+The `AssetTargetFallback` property lets you specify additional compatible target frameworks for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework`, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each target framework that's specified in `AssetTargetFallback`.
+
+You can set the `AssetTargetFallback` property to one or more target frameworks that are compatible with your project.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  ...
+  <PropertyGroup>
+    <AssetTargetFallback>netcoreapp2.0</AssetTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
 ### Pack and restore targets
 
 MSBuild 15.1 introduced `pack` and `restore` targets for creating and restoring NuGet packages as part of a build. For information about the MSBuild properties for these targets, including `PackageTargetFallback`, see [NuGet pack and restore as MSBuild targets](/nuget/reference/msbuild-targets).

--- a/docs/core/project-sdk/msbuild-props.md
+++ b/docs/core/project-sdk/msbuild-props.md
@@ -151,7 +151,7 @@ For more information, see [Package references in project files](/nuget/consume-p
 
 ### AssetTargetFallback
 
-The `AssetTargetFallback` property lets you specify additional compatible framework versions for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework` version, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each framework version that's specified in `AssetTargetFallback`.
+The `AssetTargetFallback` property lets you specify additional compatible framework versions for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework`, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each target framework that's specified in `AssetTargetFallback`.
 
 You can set the `AssetTargetFallback` property to one or more [target framework versions](../../standard/frameworks.md#supported-target-framework-versions).
 
@@ -159,7 +159,7 @@ You can set the `AssetTargetFallback` property to one or more [target framework 
 <Project Sdk="Microsoft.NET.Sdk">
   ...
   <PropertyGroup>
-    <AssetTargetFallback>netstandard1.6</AssetTargetFallback>
+    <AssetTargetFallback>net461</AssetTargetFallback>
   </PropertyGroup>
 </Project>
 ```

--- a/docs/core/project-sdk/msbuild-props.md
+++ b/docs/core/project-sdk/msbuild-props.md
@@ -132,6 +132,7 @@ For more information, see [C# language versioning](../../csharp/language-referen
 ## NuGet packages
 
 - [PackageReference](#packagereference)
+- [AssetTargetFallback](#assettargetfallback)
 
 ### PackageReference
 
@@ -150,15 +151,15 @@ For more information, see [Package references in project files](/nuget/consume-p
 
 ### AssetTargetFallback
 
-The `AssetTargetFallback` property lets you specify additional compatible target frameworks for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework`, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each target framework that's specified in `AssetTargetFallback`.
+The `AssetTargetFallback` property lets you specify additional compatible framework versions for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework` version, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each framework version that's specified in `AssetTargetFallback`.
 
-You can set the `AssetTargetFallback` property to one or more target frameworks that are compatible with your project.
+You can set the `AssetTargetFallback` property to one or more [target framework versions](../../standard/frameworks.md#supported-target-framework-versions).
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   ...
   <PropertyGroup>
-    <AssetTargetFallback>netcoreapp2.0</AssetTargetFallback>
+    <AssetTargetFallback>netstandard1.6</AssetTargetFallback>
   </PropertyGroup>
 </Project>
 ```

